<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindTheCookie</class>
 <widget class="QDialog" name="FindTheCookie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>800</width>
     <height>420</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>420</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>420</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background:rgb(0, 103, 0);
	border: 2px solid  rgb(245, 185, 125);
	border-radius: 20px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="W_cookie_crumbs_1" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="L_PX_cookie_crumbs_1">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>230</width>
           <height>340</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	border: 4px solid rgb(44, 44, 44);
	border-radius : 30px;
    background : rgb(206, 226, 242);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/images/images/cookie_crumbs.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="RB_pos1">
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Here</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/images/search.png</normalon>
         </iconset>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="RB_pos2">
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Here</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/images/search.png</normalon>
         </iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="RB_pos3">
        <property name="font">
         <font>
          <family>Javatages</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Here</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/images/search.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="W_cookie" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="L_PX_cookie">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>230</width>
           <height>340</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
    border: 4px solid rgb(44, 44, 44);
	border-radius : 30px;
    background : rgb(206, 226, 242);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/images/images/cookie_game_ftc.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="W_cookie_crumbs_2" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="L_PX_cookie_crumbs_2">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>230</width>
           <height>340</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>340</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
    border: 4px solid rgb(44, 44, 44);
	border-radius : 30px;
    background : rgb(206, 226, 242);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/images/images/cookie_crumbs.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>737</y>
     <width>961</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="PB_help_FTC">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid black;
background : rgb(186, 184, 175);
border-radius: 10px;
}

QPushButton::hover{
background : rgb(216, 214, 203);
}</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="PB_return_home_FTC">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background : rgb(198, 0, 0);
color: white;
border:1px solid blaxk;
border-radius : 10px;
}

QPushButton::hover{
background : rgb(230, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Back Home</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="L_PX_search_FTC">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>450</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/search.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <property name="bottomMargin">
     <number>50</number>
    </property>
    <item>
     <widget class="QLabel" name="L_command_panel_FTC">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Command panel</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="W_command_game_FTC" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#W_command_game_FTC{
background: rgb(245, 185, 125);
border: 2px solid green;
border-radius: 10px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="L_enter_bet">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Javatages</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter your bet : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_enter_bet">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_max_bet">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Javatages</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(max )</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_save_bet">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>American Signs</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border : 1px solid black;
border-radius : 10px;
background : rgb(170, 0, 0);
color: white;
}

QPushButton::hover{
background : rgb(148, 0, 0)
}</string>
           </property>
           <property name="text">
            <string>Save my bet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="PB_shuffle">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>American Signs</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background :  rgb(246, 170, 80);
border:1px solid black;
border-radius : 10px;
color:black;
}

QPushButton::hover
{
background  : rgb(253, 253, 0);
}
</string>
           </property>
           <property name="text">
            <string>Shuffle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_check">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>American Signs</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background : rgb(0, 103, 0);
color: white;
border:1px solid black;
border-radius : 10px;
}

QPushButton::hover{
background : rgb(0, 71, 0);
}</string>
           </property>
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Level</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QRadioButton" name="RB_level_easy">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Javatages</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>easy</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB_level_medium">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Javatages</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>MEDIUM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RB_level_hard">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Javatages</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>hard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>500</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>American Signs</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Cookies balance</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="L_cookie_balance_FTC">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="W_status_bar_FTC" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>790</y>
     <width>1021</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(245, 185, 125);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
